mhkit/tests/wave/test_contours.py

```diff
@@ -1,290 +1,585 @@
-            wave.contours.samples_contour(
-                self.te_contour, 'not an array', self.hs_contour)
-        with self.assertRaises(TypeError):
-            wave.contours.samples_contour(
-                self.te_contour, self.hs_contour, 'not an array')
-
-    def test_samples_contours_length_mismatch(self):
-        with self.assertRaises(ValueError):
-            wave.contours.samples_contour(
-                self.te_contour, self.hs_contour, np.array([1, 2]))
-
-    def test_samples_contours_range_validation(self):
-        with self.assertRaises(ValueError):
-            wave.contours.samples_contour(
-                np.array([5, 25]), self.te_contour, self.hs_contour)
-
-    def test_samples_contours_correct_interpolation(self):
+    # def test_nonparametric_copulas(self):
+    #     methods = [
+    #         "nonparametric_gaussian",
+    #         "nonparametric_clayton",
+    #         "nonparametric_gumbel",
+    #     ]
+
+    #     np_copulas = wave.contours.environmental_contours(
+    #         self.wdrt_Hm0, self.wdrt_Te, self.wdrt_dt, self.wdrt_period, method=methods
+    #     )
+
+    #     close = []
+    #     for method in methods:
+    #         close.append(
+    #             np.allclose(
+    #                 np_copulas[f"{method}_x1"],
+    #                 self.wdrt_copulas[f"{method}_x1"],
+    #                 atol=0.13,
+    #             )
+    #         )
+    #         close.append(
+    #             np.allclose(
+    #                 np_copulas[f"{method}_x2"],
+    #                 self.wdrt_copulas[f"{method}_x2"],
+    #                 atol=0.13,
+    #             )
+    #         )
+    #     self.assertTrue(all(close))
+
+    # def test_kde_copulas(self):
+    #     kde_copula = wave.contours.environmental_contours(
+    #         self.wdrt_Hm0,
+    #         self.wdrt_Te,
+    #         self.wdrt_dt,
+    #         self.wdrt_period,
+    #         method=["bivariate_KDE"],
+    #         bandwidth=[0.23, 0.23],
+    #     )
+    #     log_kde_copula = wave.contours.environmental_contours(
+    #         self.wdrt_Hm0,
+    #         self.wdrt_Te,
+    #         self.wdrt_dt,
+    #         self.wdrt_period,
+    #         method=["bivariate_KDE_log"],
+    #         bandwidth=[0.02, 0.11],
+    #     )
+
+    #     close = [
+    #         np.allclose(
+    #             kde_copula["bivariate_KDE_x1"], self.wdrt_copulas["bivariate_KDE_x1"]
+    #         ),
+    #         np.allclose(
+    #             kde_copula["bivariate_KDE_x2"], self.wdrt_copulas["bivariate_KDE_x2"]
+    #         ),
+    #         np.allclose(
+    #             log_kde_copula["bivariate_KDE_log_x1"],
+    #             self.wdrt_copulas["bivariate_KDE_log_x1"],
+    #         ),
+    #         np.allclose(
+    #             log_kde_copula["bivariate_KDE_log_x2"],
+    #             self.wdrt_copulas["bivariate_KDE_log_x2"],
+    #         ),
+    #     ]
+    #     self.assertTrue(all(close))
+
```

Comment:
should these tests still be commented? Let's remove or add back in

Reply:
Good Catch. These should be added back. They are a significant source of time during testing.